backend/app/use_case/recommendation.py

```diff
@@ -108,7 +112,15 @@ def generate_recommendation_structured_format_query(
     if trip_type:
         prompt += f"#旅行の種類\n{TRIP_TYPE_JA[trip_type]}\n"
 
-    # print("Prompt: ", prompt) # DEBUG
+    logger.info("====User query====")
+    logger.info(f"{place}")
+    logger.info(f"{date_from}")
+    logger.info(f"{date_to}")
+    logger.info(f"{people_num}")
+    logger.info(f"{budget}")
+    logger.info(f"{trip_pace}")
+    logger.info(f"{interests}")
+    logger.info(f"{trip_type}")
```

Comment:
Example of user query
```
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:115)] ====User query====
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:116)] tokyo
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:117)] 10月6日
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:118)] 10月7日
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:119)] 1
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:120)] None
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:121)] None
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:122)] None
INFO:     [2023-10-23 13:26:52 app.use_case.recommendation (recommendation.py:123)] None
```
In this case, you don't really know what query is what data so maybe add something like `f"place: {place}"`